modules/objdetect/misc/python/test/test_objdetect_aruco.py

```diff
@@ -60,7 +60,7 @@ def test_aruco_detector(self):
         gold_corners = np.array([[offset, offset],[marker_size+offset-1.0,offset],
                                  [marker_size+offset-1.0,marker_size+offset-1.0],
                                  [offset, marker_size+offset-1.0]], dtype=np.float32)
-        corners, ids, rejected = aruco_detector.detectMarkers(img_marker)
+        corners, ids, rejected, marker_unc = aruco_detector.detectMarkers(img_marker)
```

Comment:
It is better to introduce a new method overload in API to avoid breaking of existing user code.